src/tools/jsondocck/src/main.rs

```diff
@@ -50,13 +50,15 @@ pub enum CommandKind {
     Has,
     Count,
     Is,
+    HasExact,
```

Comment:
Bikeshedding: Maybe call it "HasOnly"? That makes more semantic sense to me, but not sure it makes more sense to others.